website/util/metrics.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+
+def get_entry_point(system_tags):
```

Comment:
Minor: it would be more intuitive to pass a User as the input here; that way, if we decide to store the entry point in some other way that doesn't use system_tags, we can do so without changing calls to this function.